kolibri/plugins/user_profile/assets/test/views/ChangeFacility/MergeUsernameExists.spec.js

```diff
@@ -0,0 +1,75 @@
+import { mount, createLocalVue } from '@vue/test-utils';
+import MergeUsernameExists from '../../../src/views/ChangeFacility/MergeUsernameExists';
+
+const localVue = createLocalVue();
+const sendMachineEvent = jest.fn();
+
+function makeWrapper({ targetFacility, username } = {}) {
+  return mount(MergeUsernameExists, {
+    provide: {
+      changeFacilityService: {
+        send: sendMachineEvent,
+      },
+      state: {
+        value: {
+          targetFacility,
+        },
+      },
+    },
+    mocks: {
+      $store: {
+        getters: {
+          session: { username: username },
+        },
+      },
+    },
+    localVue,
+  });
+}
+
+const getBackButton = wrapper => wrapper.find('[data-test="backButton"]');
+const getContinueButton = wrapper => wrapper.find('[data-test="continueButton"]');
+const clickBackButton = wrapper => getBackButton(wrapper).trigger('click');
+const clickContinueButton = wrapper => getContinueButton(wrapper).trigger('click');
+
+describe(`ChangeFacility/MergeUsernameExists`, () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it(`smoke test`, () => {
+    const wrapper = makeWrapper();
+    expect(wrapper.exists()).toBeTruthy();
+  });
+
+  it(`Show correct info`, () => {
+    const wrapper = makeWrapper({
+      targetFacility: { name: 'Test Facility' },
+      username: 'test1',
+    });
+    const line1_paragraph = wrapper.find('[data-test="line1"]');
```

Comment:
Would be ideal to keep the naming conventions consistent, please see below as well

Reply:
done